tests/console-backend-service/internal/domain/k8s/namespace_test.go

```diff
@@ -82,20 +94,19 @@ func givenNewTestNamespaceSuite(t *testing.T) testNamespaceSuite {
 	return suite
 }
 
+//subscribe
+func (s testNamespaceSuite) whenNamespacesAreSubscribedTo() *graphql.Subscription {
+	subscription := s.gqlClient.Subscribe(s.fixNamespacesSubscription())
```

Comment:
Thinking out loud. It's pretty dumb to send update events triggered by pods when the client is not subscribed to pods, but there's nothing we can do now.